doc/api/http.md

```diff
@@ -3567,6 +3567,59 @@ server.on('request', (request, res) => {
 server.listen(8000);
 ```
 
+## `http.createStaticServer([options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental. The implementation lacks support for partial
+> responses (Ranges) and conditional-GET negotiation (If-Match,
+> If-Unmodified-Since, If-None-Match, If-Modified-Since).
+
+* `options` {Object}
+  * `directory` {string|URL} Root directory from which files would be served.
+    **Default:** `process.cwd()`.
+  * `port` {number}
+  * `host` {string} **Default:** `'localhost'`
+  * `mimeOverrides` {Object} Dictionary linking file extension to a MIME string,
+    to override or extend the built-in ones.
+  * `filter` {Function|null} should be a function that accepts two arguments and
+    returns a value that is coercible to a {boolean} value. When `null`, no
+    files are filtered. **Default:** filters all dot files.
+  * `log` {Function|null} called when sending a response to the client.
+    **Default:** `console.log`.
```

Comment:
We have to document what the function will be called with.

Reply:
```suggestion
  * `log` {Function|null} called when sending a response to the client.
    **Default:** `console.log`.
      * `statusCode` {integer} The status code that is sent to the client.
      * `url` {string} the (origin-relative) URL that was requested.
```